lib/PasswordStrategy.js

```diff
@@ -48,7 +48,9 @@ function Strategy(options) {
             sysPassword: password
           }, function(err, verified) {
             if(err) {
-              return callback(err);
+              // ignore errors returned from verifyPassword, identities that
+              // return an error are not added to `matches`
+              return callback();
```

Comment:
Are there any errors that we actually want to surface? For example, if a user is trying to login using an identity that does have a password, but there was some database error at the time?

I guess if this happens we'll raise another issue and deal with it then. Let's just proceed and get this bug fixed.